scripts/hauler_manifest.yaml

```diff
@@ -0,0 +1,70 @@
+# This is a base file to be used to import all the required resources to run
+# Kubewarden in airgap environment into Hauler.
+#
+# To sync this file with Hauler and load everything into its storage, you can
+# run:
+# hauler store sync --filename scripts/hauler_manifest.yaml
```

Comment:
This looks like it would be a nice addition to the Makefile. We could then need to add the resulting hauler_manifest.yml as part of the GH releases.